Source/Tests/IRLoader.cpp

```diff
@@ -137,6 +137,11 @@ class DummySyscallHandler: public FEXCore::HLE::SyscallHandler {
   FEXCore::HLE::AOTIRCacheEntryLookupResult LookupAOTIRCacheEntry(uint64_t GuestAddr) override {
     return {0, 0, FHU::ScopedSignalMaskWithSharedLock {Mutex}};
   }
+
+  std::shared_mutex Mutex2;
```

Comment:
`Mutex2` isn't a descriptive name.

Reply:
It's not used there though, it's just a stub interface for the ir loader

Reply:
We probably want to get these out of the vtable tbh

Reply:
Could name it StubMutex then to imply that it is a throwaway mutex.

Reply:
Can do. The way the IRLoader works is a bit wonky wrt to the rest of our architecture, though totally not willing to rework that part right now,